lib/errors.ts

```
@@ -0,0 +1,88 @@
+    super(message, 'BLOG_POST_DATA_ERROR', cause);
+    this.slug = slug;
+  }
+}
+
+/**
+ * Utility function to wrap errors with more context
+ */
+export function wrapError<T extends AppError>(
+  message: string,
+  error: unknown,
+  ErrorClass: new (message: string, code: string, cause?: unknown) => T = AppError as any
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)